Header header
int64 detected

landmark[] landmarks3D
pose head_pose
geometry_msgs/Point gazeDirection0
geometry_msgs/Point gazeDirection1
geometry_msgs/Point gazeAngle
landmark[] landmarks2D
geometry_msgs/Point nose